src/Reducer/placeholder.js

```diff
@@ -0,0 +1,10 @@
+const placeholderReducer = (state = {}, action) =>{
```

Comment:
I have learned new stuff by going through your code. Instead of always setting our state to empty object, can't we just have an initial state object passed into our reducer?

Reply:
Thanks